engine/src/main/java/ai/hhrdr/chainflow/engine/WarehouseDelegate.java

```diff
@@ -26,10 +40,36 @@ public void execute(DelegateExecution execution) throws Exception {
             + ", variables=" + execution.getVariables()
             + " \n\n");
 
-    // Get ID, type of publication (dashboard, query, etc.) make post request to change is_draft to false
-    // Precondition: Complete is successful
-    //    execution.getVariables();
-
+    String json = "{\"is_draft\":false}";
+    HttpClient client = HttpClient.newHttpClient();
+    if (execution.getVariable("dashboard_id") != null) {
```

Comment:
switch

Reply:
switch from bpmn name, not from variables.

Reply:
Should I use getKey or getResourceName? Key has only the name for the bpmn, while resourceName includes the file extension. So far, I've written it to use the key only.